app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id, dependent: :destroy
+  has_many :comments, foreign_key: :author_id, dependent: :destroy
+  has_many :likes, foreign_key: :author_id, dependent: :destroy
+end
```

Comment:
- [ ] This module requires that the User model has a method that returns the 3 most recent posts. This should be similar to the example below:

```ruby
def update_post_counter
  author.increment!(:posts_counter)
end
```